app/controllers/api/v1/accounts_controller.rb

```diff
@@ -31,6 +31,12 @@ def create
     render json: ValidationErrorFormatter.new(e, :'account.username' => :username, :'invite_request.text' => :reason).as_json, status: :unprocessable_entity
   end
 
+  def destroy
+    authorize @account, :destroy?
+    DeleteAccountService.new.call(@account, reserve_email: false, reserve_username: false)
+    render_empty
+  end
```

Comment:
A `doorkeeper_authorize!` `before_action` is required to prevent any app from destroying the account.